src/node_file.cc

```diff
@@ -1030,26 +1031,6 @@ static void WriteString(const FunctionCallbackInfo<Value>& args) {
     SYNC_CALL(write, nullptr, fd, &uvbuf, 1, pos)
     return args.GetReturnValue().Set(SYNC_RESULT);
   }
-
-  FSReqWrap* req_wrap =
-      FSReqWrap::New(env, req.As<Object>(), "write", buf, UTF8, ownership);
```

Comment:
Just noticed that the ownership is lost below. I've fixed it locally but when writing a test for it it seems that `fs.write` does not work on external two-byte strings on master.